extensions/amp-sidebar/amp-sidebar.md

```diff
@@ -106,6 +106,12 @@ The `amp-sidebar` component can be styled with standard CSS.
 - The height of the `amp-sidebar` may be set to adjust the height of the sidebar if required. If the height exceeds 100vw then the sidebar will have a vertical scrollbar. The preset height of the sidebar is 100vw and can be overridden in CSS to make it shorter.
 - The current state of the sidebar is exposed via the `open` attribute that is set on the `amp-sidebar` tag when the side bar is open on the page.
 
+### UX Considerations
+
+When using `amp-sidebar`, bear in mind that your users will often view your page on mobile in an AMP viewer, which may display a fixed-position header. In addition, browsers often display their own fixed header at the top of the page. Adding another fixed-position element at the top of the screen would take up a large amount of mobile screen space with content that gives the user no new information.
+
+For this reason, we recommend that you do not implement your `amp-sidebar` `toggle`, `open`, or `close` affordances in a fixed layer.
+
```

Comment:
amp-sidebar is on the fixed layer by design. Also the close/toggle button is usually on the sidebar itself. So this might be the correct thing to say.